code/modules/projectiles/guns/special/syringe_gun.dm

```diff
@@ -64,7 +64,7 @@
 	user.put_in_hands(S)
 
 	syringes.Remove(S)
-	to_chat(user, span_notice("You unload [S] from \the [src]."))
+	balloon_alert(user, "[S.name] unloaded")
```

Comment:
```suggestion
	balloon_alert(user, "unloaded [S.name]")
```

Reply:
Just flows better

Reply:
For context, this comes out to "unloaded syringe" vs "syringe unloaded". I favor the latter as it's clear the former is a noun and the latter is a verb. I don't think your suggestion flows better, imo anyway. Would love somebody else to weigh in here.

Reply:
It's been 11 minutes and I decided I kinda don't care anymore